addons/vehicles/CfgVehicles.hpp

```diff
@@ -103,4 +103,118 @@ class CfgVehicles {
         crew = QCLASS(Unit_B_Contractor);
         typicalCargo[] = {QCLASS(Unit_B_Engineer)};
     };
+
+    #define MACRO_CLEAR_VEHICLE_CARGO \
```

Comment:
@jonpas Wasn't 100% on this one. Should we maybe add it as a macro to `main` instead, and use it on just about every vehicle, including optionals (BWA stuff etc.)?

Or would you rather we don't just do a macro to clear everything, but customize the inventory a bit more to fit Theseus Services (Though in our own units case, having the vehicle cargo be completely empty would be more convenient)

Reply:
I'd rather we don't clear everything, but modify accordingly. 

For Theseus itself, we would have to do it for every vehicle, including vanilla anyways.

Reply:
Maybe just add only 4 First Aid Kits (Similar to the Offroad currently)?

Reply:
That sounds good.